### DISCUSSION TITLE
Only Build the QT GUI part with  QT 6.6, but agents.h not found

### DISCUSSION THREAD

**Discussion by zhiyoutech** [x64dbg/x64dbg#3105] [General]
Hi,  nice to talk with you.

I want to build the GUI alone without other part. I cannot find agents.h in the building.

I think the version of GUI which is 5.6.3 is a little old. I tried many times to build QT GUI with QT6.6, but it is failed.
Could you please update the version of QT to be greater than 6.2, and make an easier  building lesson only with GUI?

many thanks!

**Comment by torusrxxx** [comment]
#3112

**Reply by zhiyoutech** [reply]
Thanks!  That is very good!
